src/tigerbeetle.zig

```diff
@@ -84,7 +84,9 @@ pub const TransferFlags = packed struct {
     pending: bool = false,
     post_pending_transfer: bool = false,
     void_pending_transfer: bool = false,
-    padding: u12 = 0,
+    debits_at_most: bool = false,
+    credits_at_most: bool = false,
```

Comment:
Since `debits` or `credits` is the qualifier, and `at_most` is the feature, it would be better to swap these around, also so that the feature name lines up.

The `at_most` could then be renamed to `closing` in the sense of a closing transfer, to give:
* `closing_debit`
* `closing_credit`

Reply:
We can then call these Closing Transfers, and the docs can explain how these work in more detail (the "at most" part), whereas `debits_at_most` is not a noun so it's hard to refer to in verbal communication.

Reply:
I chatted with @lewisdaly at the office today and he grokked what `balancing_debit/credit` might mean.